recipes/instaseis/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "7fece415cca6c70e575531a463c0d5cb6d30f59740d7a113a02d584775a8b0c1" %}
+
+package:
+  name: instaseis
+  version: {{ version }}
+
+
+source:
+  fn: instaseis-{{ version }}.zip
+  url: https://pypi.io/packages/source/i/instaseis/instaseis-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc
+  run:
+    - python
+    - libgfortran
+    - numpy
+    - h5py
+    - obspy
+    - future
+    - tornado
+    - requests
+    - jsonschema
+    - pyqt
+    - pyqtgraph
+
+test:
+  imports:
+    - instaseis
+  commands:
```

Comment:
This is just the Mac OS default. For instance on CentOS 6 I find it is `1048576`. IMHO Apple is being way too cautious.

Though I understand the desire not to muck with these system defaults. The place to solve this will be in the test suite itself.

Personally I think you are ok making this sort of limit change for now. Maybe you will want to experiment to more carefully determine how large it can be.